Transfer/Sources/Extensions/HyperwalletTransferUI.swift

```diff
@@ -22,18 +22,34 @@ import Common
 
 /// The HyperwalletUI extension
 public extension HyperwalletUI {
-    /// Create transfer funds
+    /// Lists the user's transfer methods (bank account, bank card, PayPal account, prepaid card, paper check).
```

Comment:
Why update the comment? This method is to create transfer funds from user and not to list transfer methods.

Reply:
Changed the comments